src/configure/helper/remoteServiceUrlHelper.ts

```diff
@@ -26,10 +29,19 @@ export class RemoteServiceUrlHelper {
             } as IServiceUrlDefinition;
         }
 
-        return this.getServiceurlDefinition("https://peprodscussu2.portalext.visualstudio.com", this.templateServiceRedirectUrl);
+        if (deployment != undefined && deployment === "test") {
```

Comment:
should we name it "staging" instead of "test"?

Reply:
done